src/officehours_api/serializers.py

```diff
@@ -228,6 +228,13 @@ def validate_host_ids(self, host_ids):
         else:
             return host_ids
 
+    def validate_allowed_backends(self, value):
```

Comment:
I cannot find the usage of this new validate_allowed_backends() or the validate_backend_type() function in the code base.

Reply:
@jonespm Is the new validate_allowed_backends() used anywhere in the code?

Reply:
This is a [special DRF pattern](https://www.django-rest-framework.org/api-guide/serializers/#field-level-validation
) for `validate_<field_name>` so they're used from the library. 

I can see if I can provide a unit test or test case for it to validate that it's working and useful. I did check in a debugger that it was being called but all of the values were always valid.